crates/bevy_render/src/render_resource/shader.rs

```diff
@@ -172,12 +172,7 @@ impl ProcessedShader {
                     let wgsl = reflection.get_wgsl()?;
                     ShaderSource::Wgsl(wgsl.into())
                 }
-                ProcessedShader::SpirV(_) => {
-                    // TODO: we can probably just transmute the u8 array to u32?
-                    let reflection = self.reflect()?;
-                    let spirv = reflection.get_spirv()?;
-                    ShaderSource::SpirV(Cow::Owned(spirv))
-                }
+                ProcessedShader::SpirV(source) => make_spirv(source),
```

Comment:
Does this function support spirv, wgsl and glsl inputs?

Reply:
probably not https://docs.rs/wgpu/latest/wgpu/util/fn.make_spirv.html

Reply:
Indeed. According to the source it only allows spirv.